### DISCUSSION TITLE
Github Project beta - drafts do not respond to scrolling

### DISCUSSION THREAD

**Discussion by pritamKarmokar** [community/community#15092] [Projects and Issues]
I just started using the Github Project beta smart spreadsheets. Few of the draft item markdowns I created had longer bodies. For such longer drafts the complete content is only seen in the `edit` mode, but not in the `preview` mode. Also, few drafts had wide tables but the preview is too narrow and without horizontal or vertical scrollbars; non-responsive to scrolling gestures or mouse wheel actions.
It is super inconvenient and undesirable to be unable to view the complete markdown draft doc.
I love the Project beta tool, and would like to start using it on a daily basis. I would really appreciate it if this issue gets a fix as soon as possible. Cheers!

**Comment by Sammeeey** [comment]
I think that's the view which you mean, right?:
![grafik](https://user-images.githubusercontent.com/49591562/164085988-6a94e1c7-6ffb-4988-962b-a4ba9002c701.png)

I can't scroll there either.

**Reply by pritamKarmokar** [reply]
Yes, @Sammeeey, that is correct; thank you for the screenshot. Just like in your screenshot, I too had long draft bodies, which when viewed (imho) needs responsiveness to scrolling.
